app/Filament/Resources/PatientHistoryResource.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace App\Filament\Resources;
+
+use App\Filament\Resources\PatientHistoryResource\Pages;
+use App\Filament\Resources\PatientHistoryResource\RelationManagers;
+use App\Models\PatientHistory;
+use Filament\Forms;
+use Filament\Forms\Components\DatePicker;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\Textarea;
+use Filament\Forms\Components\TextInput;
+use Filament\Forms\Form;
+use Filament\Resources\Resource;
+use Filament\Tables;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Filters\SelectFilter;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\SoftDeletingScope;
+
+class PatientHistoryResource extends Resource
+{
+    protected static ?string $model = PatientHistory::class;
+
+    protected static ?string $navigationGroup = 'Patients';
+    protected static ?string $navigationIcon = 'heroicon-o-rectangle-stack';
+
+    public static function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                Select::make('patient_id')
+                    ->relationship('patient', 'name')
+                    ->native(false)
+                    ->required(),
+                DatePicker::make('date')
+                    ->required()
+                    ->default(today())
+                    ->native(false),
+                TextInput::make('title')
+                    ->required(),
+                Textarea::make('food_allergies')
+                        ->maxLength(225),
+                Textarea::make('bleed_tendency')
+                    ->maxLength(225),
+                Textarea::make('heart_disease')
+                    ->maxLength(225),
+                Textarea::make('blood_pressure')
+                    ->maxLength(225),
+                Textarea::make('diabetic')
+                    ->maxLength(225),
+                Textarea::make('surgery')
+                    ->maxLength(225),
+                Textarea::make('accident')
+                    ->maxLength(225),
+                Textarea::make('family_medical_history')
+                    ->maxLength(225),
+                Textarea::make('current_medication')
+                    ->maxLength(225),
+                Textarea::make('female_pregnancy')
+                    ->maxLength(225),
+                Textarea::make('breast_feeding')
+                    ->maxLength(225),
+                Textarea::make('health_insurance')
+                    ->maxLength(225),
+                Select::make('low_income')
+                    ->options([
+                        0 => 'No',
+                        1 => 'Yes'
+                    ])
+                ->native(false),
+                Textarea::make('reference')
+                    ->maxLength(225),
+                Textarea::make('others')
+                    ->maxLength(225),
+                Select::make('status')
```

Comment:
@kolanaveen Let's make this field `required` because if I haven't select any value then it is giving error.

Reply:
I've chosen a default value for the input, and I've also set it in the migration.

Reply:
@kolanaveen Yes, you have set default value but if i remove those default value and save changes then it is giving error.

Reply:
@DivyaPatange OK fixed.